Include/internal/pycore_runtime_generated.h

```diff
@@ -0,0 +1,1138 @@
+#ifndef Py_INTERNAL_RUNTIME_GENERATED_H
```

Comment:
It may be a bit more clear if the file is named "pycore_runtime_init_generated.h".